messaging/src/main/java/org/axonframework/eventhandling/SubscribingEventProcessor.java

```diff
@@ -215,6 +218,12 @@ public Builder messageMonitor(@Nonnull MessageMonitor<? super EventMessage<?>> m
             return this;
         }
 
+        @Override
+        public Builder spanFactory(@Nonnull EventProcessorSpanFactory spanFactory) {
+            super.spanFactory(spanFactory);
+            return this;
+        }
+
```

Comment:
Shouldn't this change deprecate the other builder?

Reply:
Correct, done